standard.md

```diff
@@ -0,0 +1,114 @@
+- **log_level**: `str`, default values is `info`. Sets the granularity of the diagnostic logs and console output. If used, diagnostic must accept at least `error` `warning`, `info` and `debug`
+- **auxiliary_data_dir** : `str`. Path to the auxiliary directory where the diagnostic may find further input data.
+- **max_proc_number** : `integer`, default value is **1**. The maximum number of processors that the tool allows the diagnostic to request
+
+
+## The data definition file
+
+The data definition file must be a yaml file containing a dictionary of dictionaries. This first dictionary will have as keys the path to the data files and as values the dictionary containg the metadata of each file.
+
+There is no limit to the number of file that can be provided on a single file, although it is suggested
+
+### Required options
+
+- **alias**: `str`. Unique name for the dataset. Any (`alias`, `variable`) pair must be unique
+- **filename**: `str`. Absoulute path to the data file.
+- **variable**: `str`. Variable name to be used by the diagnostic. It does not have to match any of the CF conventions names in the file
+
+### Reserved options
+
+#### Related to the dataset
+
+- **project**: `str`. Name of the project at which this dataset belongs. It can be a real project, like `CMIP5`, `CMIP6` or a tool defined agrupation of datasets like `Observations` or `Reanalysis`
+- **activity**: `str`. CMIP activity or analog for the current data
+- **institute**: `list(str)`. List of institutes associated to the current data.
+- **dataset**: `str`. Name of the dataset, being it the CMIP model indetifier, the name of an observation
+- **ensemble**: `str`. Ensemble identifier for the current data
+
+#### Related to the variable
+
+- **table**: `str`. Original mip of the current variable
+- **frequency**: `str`. Original frequency of the data used
+- **modeling_realm**: `list(str)`. List of realms at which the variable belonh
+- **grid**: `str`. Original grid of the variable
+- **units**: `str`. Units of the data
+- **short_name**: `str`. Variable name used in the file.
+- **standard_name**: `str`. Standard name of the variable.
+- **long_name**: `str`. Long name of the variable
+
+#### Others
+
+- **start**: `str`. Starting date of the data, in the format YYYYMMDD
+- **end**: `str`, Ending date of the data, in the format YYYYMMDD
+- **start_year**: `int`. Starting year of the data
+- **end_year**: `int`. End year of the data
+- **reference_dataset**: `str`. Dataset to be used as the reference to compare the current dataset with, given as the alias.
+
+## The script formal description file
+
+It is recommended that each script comes with a description file in yaml syntax which includes :
+- **script_name**: `str`. A label for the name of the script, which uniquely identifies it, and is delivered by an authoritative entity (such as an ESGF, ENES or WCRP entitiy)
```

Comment:
```suggestion
- **script_name**: `str`. A label for the name of the script, which uniquely identifies it, and is delivered by an authoritative entity (such as an ESGF, ENES, or WCRP entity)
```
Is there a provider of such names? This will be difficult to implement without one of those.